www/localPackage.ts

```diff
@@ -107,8 +107,19 @@ class LocalPackage extends Package implements ILocalPackage {
     }
 
     private finishInstall(deployDir: DirectoryEntry, installOptions: InstallOptions, installSuccess: SuccessCallback<InstallMode>, installError: ErrorCallback): void {
+        function backupPackageInformationFileIfNeeded(backupIfNeededDone: Callback<void>) {
+            NativeAppInfo.isPendingUpdate((pendingUpdate: boolean) => {
+                if (pendingUpdate) {
+                    // Don't back up the  currently installed update since it hasn't been "confirmed"
+                    backupIfNeededDone(null, null);
```

Comment:
The callback backupIfNeededDone seems to only take one argument. Why are you passing 2 nulls here?